scalalib/src/dependency/versions/Version.scala

```diff
@@ -47,9 +47,9 @@ case class ValidVersion(
 ) extends Version {
   def major: Long = releasePart.headOption getOrElse 0
 
-  def minor: Long = releasePart.drop(1).headOption getOrElse 1
+  def minor: Long = releasePart.drop(1).headOption getOrElse 0
 
-  def patch: Long = releasePart.drop(2).headOption getOrElse 1
+  def patch: Long = releasePart.drop(2).headOption getOrElse 0
```

Comment:
Defaulting to `1` seems like a bug from the beginning. At least I can't come up with why this may be the right thing.